x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -119,6 +122,19 @@ export default function (ctx: FtrProviderContext) {
         await globalNav.badgeMissingOrFail();
       });
 
+      it('Shows short urls for the right priviledged user', async () => {
```

Comment:
```suggestion
      it('Shows short urls for users with the right privileges', async () => {
```
Nit: I'd make this description consistent with the same one below.